recipes/cudatoolkit/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "cudatoolkit" %}
+{% set version = "9.2.88_396.26" %}
+{% set major_version = '.'.join(version.split('.')[:2]) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://developer.nvidia.com/compute/cuda/{{ major_version }}/Prod/local_installers/{{ "cuda_" + version + "_linux" }}  # [ linux ]
+    sha256: 8d02cc2a82f35b456d447df463148ac4cc823891be8820948109ad6186f2667c  # [ linux ]
+    fn: {{ "cuda_" + version + "_linux" }}  # [ linux ]
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - perl  
+
+test:
+  requires:
+    - numba
+
+about:
+  home: https://developer.nvidia.com
+  license: other
+  license_family: other
+  license_file: '{{ environ["RECIPE_DIR"] }}/EULA.pdf'
+  summary: 'Develop, Optimize and Deploy GPU-accelerated Apps'
+  description: |
+    The NVIDIA CUDA Toolkit provides a development environment for creating 
+    high performance GPU-accelerated applications. With the CUDA Toolkit, 
+    you can develop, optimize and deploy your applications on GPU-accelerated
+    embedded systems, desktop workstations, enterprise data centers, 
+    cloud-based platforms and HPC supercomputers. The toolkit includes 
+    GPU-accelerated libraries, debugging and optimization tools, 
+    a C/C++ compiler and a runtime library to deploy your application.
+    This package just have some binaries from the full package. 
+  doc_url: https://developer.nvidia.com/cuda-toolkit
+  dev_url: https://developer.nvidia.com/cuda-toolkit
+
+extra:
+  recipe-maintainers:
+    - xmnlab
+    - scopatz
+    - andersy005
```

Comment:
Thanks @xmnlab! @andersy005 are you OK with being a maintainer?

Reply:
@scopatz, Yes!